com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/GraphicsToolsStandard.shader

```diff
@@ -184,11 +184,57 @@ Shader "Graphics Tools/Standard"
 
             #pragma shader_feature_local_fragment _CLIPPING_BORDER
 
+            // From SimpleLit SubShader "ForwardLit"...
+            // ...
+            // -------------------------------------
+            // Universal Pipeline keywords
+            #pragma multi_compile _ _MAIN_LIGHT_SHADOWS
+            #pragma multi_compile _ _MAIN_LIGHT_SHADOWS_CASCADE
+            #pragma multi_compile _ _ADDITIONAL_LIGHTS_VERTEX _ADDITIONAL_LIGHTS
+            #pragma multi_compile_fragment _ _ADDITIONAL_LIGHT_SHADOWS
+            #pragma multi_compile_fragment _ _SHADOWS_SOFT
+            #pragma multi_compile _ LIGHTMAP_SHADOW_MIXING
+            #pragma multi_compile _ SHADOWS_SHADOWMASK
+            #pragma multi_compile_fragment _ _SCREEN_SPACE_OCCLUSION
```

Comment:
Are these keywords used anywhere yet?

Reply:
This is similar to below. Commenting out these features have no impact on appearance of shadows. 🤔Strangely, the built-in Simple Lit doesn't not appear to support multiple light shadows either. I may have a configuration issue?

![image](https://user-images.githubusercontent.com/538256/213017177-0ef1b78c-45ed-4887-9229-c43e5639ca31.png)